webpack.common.js

```diff
@@ -127,6 +127,14 @@ const CommonConfig = {
           },
         ],
       },
+      {
+        test: /\.mjs$/,
+        include: /node_modules/,
+        type: 'javascript/auto',
+        resolve: {
+          fullySpecified: false
+        }
+      },
```

Comment:
Resolve this following error:

```
ERROR in ./node_modules/axios/lib/utils.js 698:42-49
--
19-Aug-2024 13:34:51 | Module not found: Error: Can't resolve 'process/browser' in '/data/bamboo/bamboo-agent-home/xml-data/build-dir/CUM-CDG239-CUBPR/cumulus-dashboard/node_modules/axios/lib'
19-Aug-2024 13:34:51 | Did you mean 'browser.js'?
19-Aug-2024 13:34:51 | BREAKING CHANGE: The request 'process/browser' failed to resolve only because it was resolved as fully specified
19-Aug-2024 13:34:51 | (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
19-Aug-2024 13:34:51 | The extension in the request is mandatory for it to be fully specified.
19-Aug-2024 13:34:51 | Add the extension to the request
```